src/style.css

```diff
@@ -0,0 +1,156 @@
+/* stylelint-disable-next-line no-empty-source */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  --main-text-color: #060b35;
+}
+
+.product {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  border: rgb(245, 245, 245) solid 0.5px;
+  border-radius: 5px;
+  max-width: 199px;
+  max-height: 407px;
+}
+
+.title {
+  display: flex;
+  margin-top: 40px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 4px;
+}
+
+.product_title {
+  color: var(--main-text-color);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  padding-left: 0.5px;
+}
+
+.image {
+  display: flex;
+  margin-top: 32px;
+  margin-left: 19px;
+  margin-right: 19px;
+}
+
+.product_image {
+  height: 134px;
+  width: 160px;
+}
+
+.product_code {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+
+  margin-left: 16px;
+  margin-bottom: 16px;
+}
+
+.pcode_text {
+  font-family: "Roboto", sans-serif;
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  padding-left: 0.5px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  padding-left: 0.5px;
+}
+
+.stars__start {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 12px;
+  height: 12px;
+  padding: 0 7.5px 0 0;
+  margin-left: 0.5px;
```

Comment:
I think you can add background-position: center rule and remove this margin and backgrouind-size rules